recipe/meta.yaml

```diff
@@ -18,33 +18,42 @@ build:
 
 requirements:
   build:
-    - posix  # [win]
     - {{ compiler('c') }}  # [not win]
-    - {{ compiler('m2w64_c') }}  # [win]
     - {{ compiler('cxx') }}  # [not win]
+    # http://pystan.readthedocs.io/en/latest/windows.html
+    - {{ compiler('m2w64_c') }}  # [win]
     - {{ compiler('m2w64_cxx') }}  # [win]
   host:
     - python
-    - setuptools
+    - pip
     - numpy
     - cython >=0.22,!=0.25.1
+    - scipy
+    - setuptools
+    - libpython  # [win]
   run:
-    # http://pystan.readthedocs.io/en/latest/windows.html
-    # PyStan requires a C++ compiler for building models. For
-    # full support of vectorized functions, gcc is recommended.
-    - posix  # [win]
-    - {{ compiler('c') }}  # [not win]
-    - {{ compiler('m2w64_c') }}  # [win]
-    - {{ compiler('cxx') }}  # [not win]
-    - {{ compiler('m2w64_cxx') }}  # [win]
-    - python
```

Comment:
We need to keep these in the run requirements, right? Otherwise users won't be able to build their stand models if they install pystan out of the box.

Reply:
If conda was having trouble making the run env, then it might be due to a bug in version 4.7.11 and version 4.7.12. We could try to force version 4.7.10 for builds.